lms/djangoapps/branding_by_referer/branding_by_referer.py

```diff
@@ -119,22 +123,26 @@ def process_response(self, request, response):
         """
         if not self.check_feature_enable():
             return response
+
         if self.pending_cookie:
             self.set_cookie(response, self.pending_cookie)
             self.pending_cookie = None
 
-        if SetBrandingByReferer.user_referer:
+        current_branding_by_referer = getattr(get_current_request(), 'branding_by_referer', {})
```

Comment:
A question here: the request parameter passed to the process_response method is equal to the result of the method get_current_request? can we use the request object directly instead of calling to get_current_request?

Reply:
@Squirrel18 please let me know if my question here had no sense